Template/template.json

```diff
@@ -1011,6 +1011,45 @@
         }
       ]
     },
+    {
+      "categories": ["Web","Tools"],
+      "description": "Git with a cup of tea! Painless self-hosted all-in-one software development service, including Git hosting, code review, team collaboration, package registry and CI/CD.",
+      "env": [
+        {
+          "default": "1000",
+          "label": "PUID",
+          "name": "PUID"
+        },
+        {
+          "default": "100",
```

Comment:
Shouldn't this be `1000`?

Reply:
Correct, I'll open a pr to fix it.

Reply:
- https://docs.linuxserver.io/general/understanding-puid-and-pgid/
- https://www.reddit.com/r/docker/comments/tshn82/how_does_puidpgid_actually_work_in_docker/
I wish I could find a more official reference, but it seems the pattern is to reference `1000` for both `PUID` and `PGID`.

It could be that `100` is a remnant of copy-paste.
And it could be there was once an image that required the `PGID` be set to `100`. But I think most docker containers would be best set to `1000`.

I went ahead and created a PR (https://github.com/Qballjos/portainer_templates/pull/70), just in case you change your mind.

Reply:
Say if I'm wrong but wouldn't changing it on systems that have already deployed containers possibly result in permission issues?

Reply:
I think it's safe to call it a backwards-breaking change.

If you create a V3 file, that might be the best chance to update the PGID for these existing templates.

Reply:
For this reason, I'll close my PR